src/line_extractor_igvc/test/line_extractor_test.test

```diff
@@ -0,0 +1,11 @@
+<!-- A .test file which launches the node and its corresponding rostest -->
```

Comment:
We should update the default description here.

Reply:
Done